app/src/main/java/com/nutomic/syncthingandroid/service/RunConditionMonitor.java

```diff
@@ -16,19 +16,31 @@
 import android.os.Build;
 import android.os.PowerManager;
 import android.support.annotation.Nullable;
-import android.support.v4.content.LocalBroadcastManager;
 import android.util.Log;
 
-import com.google.common.collect.Lists;
+import com.nutomic.syncthingandroid.R;
 import com.nutomic.syncthingandroid.SyncthingApp;
-import com.nutomic.syncthingandroid.service.ReceiverManager;
+import com.nutomic.syncthingandroid.model.RunConditionCheckResult;
 
+import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 import javax.inject.Inject;
 
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.GLOBAL_SYNC_DISABLED;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.NO_ALLOWED_NETWORK;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.NO_MOBILE_CONNECTION;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.NO_NETWORK_OR_FLIGHTMODE;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.NO_WIFI_CONNECTION;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.ON_BATTERY;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.ON_CHARGER;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.POWERSAVING_ENABLED;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.WIFI_SSID_NOT_WHITELISTED;
+import static com.nutomic.syncthingandroid.model.RunConditionCheckResult.Reason.WIFI_WIFI_IS_METERED;
```

Comment:
I think you can import the enum as a whole, and not list every item separately? Not sure, my Java is a bit rusty.

Otherwise looks good 👍

Reply:
Could be, it was generated that way by Android Studio. I'll check.